packages/browser-logs/src/deserialize.ts

```diff
@@ -41,7 +41,12 @@ function createReviver(promises: Promise<unknown>[], options?: DeserializeOption
           return;
         case 'Function':
           // Create a fake function with the same name. We don't log the function implementation.
-          return new Function(`return function ${value.name}() { /* implementation hidden */ }`)();
+          return new Function(
+            `return function ${value.name.replace(
+              /^bound\s+/,
```

Comment:
Can we make the regexp a constant? That avoids recreating on each call, since this code is run a lot it does make a difference.

Reply:
I pushed the change